presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixConfig.java

```diff
@@ -59,6 +59,7 @@ public static ReadMode fromString(String value)
     private String cacheLocation;
     private int bookKeeperServerPort = CacheConfig.DEFAULT_BOOKKEEPER_SERVER_PORT;
     private int dataTransferServerPort = CacheConfig.DEFAULT_DATA_TRANSFER_SERVER_PORT;
+    private boolean startServerOnCoordinator;
```

Comment:
This means that we no longer need cache directory on coordinator if this property is set to false.
Can you please update documentation accordingly?

Reply:
Cache directories are still validated in `setupServer`, so it's still required to have cache directory on coordinator